fossunited/api/schedule.py

```diff
@@ -94,3 +96,30 @@ def get_schedule_by_hall(schedule: list) -> dict:
         _schedule[hall].append(session)
 
     return _schedule
+
+
+@frappe.whitelist(allow_guest=True)
+def get_schedule_session_ics(schedule: dict):
```

Comment:
Tangential - but this method could prove to be generic to add an Add to Calendar Integration for an event as a whole. 

Only if it's decided to make it as an integration in the event page, as referred in #585.

Reply:
I will create a generic one when I am working on it. Don't see the need to overcomplicate things at this point in time tbh

Reply:
Refer this too - https://github.com/fossunited/fossunited/pull/584#discussion_r1736895097